pytensor/link/pytorch/dispatch/slinalg.py

```diff
@@ -0,0 +1,28 @@
+import torch.linalg
+
+from pytensor.link.pytorch.dispatch import pytorch_funcify
+from pytensor.tensor.slinalg import Cholesky, SolveTriangular
+
+
+@pytorch_funcify.register(Cholesky)
+def pytorch_funcify_Cholesky(op, **kwargs):
+    lower = op.lower
+
+    def cholesky(a, lower=lower):
+        return torch.linalg.cholesky(a, upper=not lower)
+
+    return cholesky
+
+
+@pytorch_funcify.register(SolveTriangular)
+def pytorch_funcify_SolveTriangular(op, **kwargs):
+    lower = op.lower
+    trans = op.trans
+    unit_diagonal = op.unit_diagonal
+
+    def solve_triangular(A, b):
+        return torch.linalg.solve_triangular(
+            A, b, upper=not lower, unit_triangle=unit_diagonal, left=trans == "T"
```

Comment:
trans may also be `1` apparently. We should canonicalize the integers to strings so the Op only has to handle one of the encodings: https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.solve_triangular.html